CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -128,33 +145,51 @@ def main() -> int:
         mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
         modrinth_config = git_path + "Modrinth\\mmc-export.toml"
 
-        cmd = f'mmc-export -i "{mmc_zip_path}" -f Modrinth --modrinth-search loose -o "{mmc_zip_root}" -c "{modrinth_config}" -v {pack_version} --scheme {"{name}-{version}"}'
-        os.system(cmd)
-
-        if is_legacy == False:
-            extract_file(mmc_zip_root + "\\Fabulously Optimized-" + pack_version + ".mrpack", "modrinth.index.json", git_path + "\\" + "Modrinth", "Modrinth manifest", "Git")
-
-    # Export Modrinth pack and manifest via packwiz method
+        os.system(
+            " ".join(
+                (
+                    "mmc-export",
+                    f'-i "{mmc_zip_path}"',
+                    "-f Modrinth",
+                    "--modrinth-search loose",
+                    f'-o "{mmc_zip_root}"',
+                    f'-c "{modrinth_config}"',
+                    f"-v {pack_version}",
+                    f'--scheme {"{name}-{version}"}',
+                )
+            )
+        )
+
+        if not is_legacy:
+            extract_file(
+                mmc_zip_root + "\\Fabulously Optimized-" + pack_version + ".mrpack",
+                "modrinth.index.json",
+                git_path + "\\" + "Modrinth",
+                "Modrinth manifest",
+                "Git",
+            )
+
+    # Export Modrinth pack and manifest via Packwiz method
     if packwiz_modrinth_export:
-        os.system(packwiz_exe_path + " modrinth export")
+        os.system(f"{packwiz_exe_path} modrinth export")
         for pack in os.listdir(packwiz_path):
-            if pack.endswith('.mrpack'):
-                if is_legacy == False:
-                    extract_file(packwiz_path + "\\" + pack, "modrinth.index.json", git_path + "\\" + "Modrinth", "Modrinth manifest", "Git")
+            if pack.endswith(".mrpack"):
+                if not is_legacy:
+                    extract_file(
+                        packwiz_path + "\\" + pack,
+                        "modrinth.index.json",
+                        git_path + "\\" + "Modrinth",
+                        "Modrinth manifest",
+                        "Git",
+                    )
                 os.replace(packwiz_path + "\\" + pack, os.path.expanduser("~/Desktop") + "\\" + pack)
-                print("Moved " + pack + " to desktop")
-        os.system(packwiz_exe_path + " refresh")
-    
-    if not refresh_only:

```

Comment:
The goal of this section is removing jars from the MMC zip, if CF decides to upload a mod and archive it in its site. While I haven't used it for a while, I may still need that in the future.

Reply:
> The goal of this section is removing jars from the MMC zip, if CF decides to upload a mod and archive it in its site. While I haven't used it for a while, I may still need that in the future.

OK, brought it back in [72d439b](https://github.com/Fabulously-Optimized/fabulously-optimized/pull/754/commits/72d439bd22d147aa2e4a880db7187ffab6e28ad2)